satellite/partners/partners.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package partners
```

Comment:
Does this need to be in its own package? @coyle

Reply:
I tried keeping it in `marketingweb` but it throws this error since `rewards` is calling the load function in it.

<img width="568" alt="Screen Shot 2019-07-17 at 3 01 09 PM" src="https://user-images.githubusercontent.com/34631293/61407835-aeae2380-a8ac-11e9-9ede-6da5e2194bf1.png">

Reply:
Would it be ok to have it under `rewards` package?